### ISSUE TITLE
Keyboard Focus Cannot Leave Tab

### ISSUE THREAD

**Issue by getBoolean** [getBoolean/flutter_boolean_template#101]
**Describe the bug**
https://github.com/Milad-Akarie/auto_route_library/issues/1189

**Additional context**
Seeing the same issue with `AutoTabsScaffold` and `AutoTabsRouter`. It may have something to do with tab routes. Still happening after switching to `go_router`

**Comment by getBoolean** [comment]
flutter issue, should be fixes in the next stable 3.19 release

**Comment by getBoolean** [comment]
switched app to beta channel, so closing this
